gui/src/components/SettingsNetwork.tsx

```diff
@@ -64,15 +62,13 @@ export function SettingsNetwork() {
   });
 
   const onSubmit = async (data: { networks?: Network[] }) => {
-    await invoke("set_networks", { networks: data.networks });
+    if (data.networks) await setNetworks(data.networks);
```

Comment:
shouldn't this guard cover the `reset` as well?

(I'm not sure when this would actually happen)

Reply:
oh, yeah 👍